content/docs/spec.md

```diff
@@ -120,13 +120,14 @@ The server MUST wait until C0 has been received before sending S0 and S1, and MA
 #### 5.2.2.	C0 and S0 Format
 
 The C0 and S0 packets are a single octet, treated as a single 8-bit integer field:
-```
+
+<figure><pre>
  0 1 2 3 4 5 6 7
 +-+-+-+-+-+-+-+-+
 |    version    |
 +-+-+-+-+-+-+-+-+
  C0 and S0 bits
```

Comment:
prolly want `C0 and S0 bits` to be figcaption?

Reply:
I wasn't sure if those were meant to be captions or not when I read the original PDF. Looking again, I think you're right -- they were meant to be captions. I'll update and push a fix. Thx for catching!